### DISCUSSION TITLE
Bind Avalonia Control to SQLite Database Table

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#5564] [Q&A] [answered]
Hi,
I am working on an application where all the data is stored in an in-memory SQLiteDatabase.
Currently I am loading all the data to an ObservableCollection and binding it with the Grid. Is there a way to bind a DataGrid control directly to the database table?

**Comment by maxkatz6** [comment, marked as answer]
No, currently Avalonia DataGrid doesn't work with DataTable class.
